src/main/java/com/minecolonies/inventory/InventoryCitizen.java

```diff
@@ -656,6 +651,14 @@ public void openInventory(EntityPlayer player)
         /*
          * This may be filled in order to specify some custom handling.
          */
+    }    /**
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)